### ISSUE TITLE
addons/knobs types were unpublished

### ISSUE THREAD

**Issue by benbayard** [storybooks/storybook#1558]
After updating from v3.1.9 to v3.2.0 the typescript types were no longer published on npm.

**Comment by shilman** [comment]
@benbayard yeah we could have done a better job documenting that. Here's the MIGRATION instructions:

https://github.com/storybooks/storybook/blob/master/MIGRATION.md#from-version-31x-to-32x

And here's the relevant github issue: https://github.com/storybooks/storybook/issues/1166

We'll try to get those types updated today (cc @joscha )

**Comment by benbayard** [comment]
Thanks @shilman ! If you guys are interested in using TS as the language maintained for the knobs repo I can work on that.

**Comment by shilman** [comment]
@benbayard Thanks! I'm sure we could use some help on the Typescript front. @joscha is handling that now, so I'll let you guys coordinate. Easiest way to coordinate (IMHO) is our slack channel:

https://now-examples-slackin-nqnzoygycp.now.sh/

**Comment by ndelangen** [comment]
@benbayard That would be fantastic, we could definitely use the help!
